table_engine/src/provider.rs

```diff
@@ -253,7 +253,7 @@ impl TableSource for TableProviderAdapter {
     /// Tests whether the table provider can make use of a filter expression
     /// to optimize data retrieval.
     fn supports_filter_pushdown(&self, _filter: &Expr) -> Result<TableProviderFilterPushDown> {
-        Ok(TableProviderFilterPushDown::Inexact)
```

Comment:
L229 also need to update.

Reply:
it will no pass this test case, I may need to find other things that need fix🤔
https://github.com/CeresDB/ceresdb/blob/62d38355311af987ad40e8fd5dbca4d163e19d70/integration_tests/cases/common/dml/select_filter.sql#L24-L32

Reply:
Only this case will failed?

Reply:
and this use case is wrong
https://github.com/CeresDB/ceresdb/blob/7a701c6df8f687babeb5602dbc88c7ead07fa4a8/integration_tests/cases/env/local/system/system_tables.sql#L14-L23

Reply:
OK.

I will also investigate why those cases fails next week, you can try other issue first in you think this is too hard for you.

Reply:
maybe I can try to fix the problem this weekend. 😥